### ISSUE TITLE
[Bug]: 400 error when hitting Claude-3.5 via Bedrock using bedrock-httpx client

### ISSUE THREAD

**Issue by ubirdio** [BerriAI/litellm#4487]
### What happened?

I just upgraded litellm from 1.35.38 to 1.41.2.
My calls to `bedrock/anthropic.claude-3-5-sonnet-20240620-v1:0` now fail with the error `extraneous key [max_tokens_to_sample]`.

I have tried to narrow down the relevant version and it seems to be specifically 1.41.2. `pip install litellm==1.41.1` and it works, but at `1.41.2` it breaks.

For now I'm reverting to 1.41.1 to deal with this.

### Relevant log output

```shell
Traceback (most recent call last):
  File "/path/to/venv/lib/python3.11/site-packages/litellm/llms/bedrock_httpx.py", line 1123, in async_completion
    response = await client.post(api_base, headers=headers, data=data)  # type: ignore
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/venv/lib/python3.11/site-packages/litellm/llms/custom_httpx/http_handler.py", line 122, in post
    raise e
  File "/path/to/venv/lib/python3.11/site-packages/litellm/llms/custom_httpx/http_handler.py", line 99, in post
    response.raise_for_status()
  File "/path/to/venv/lib/python3.11/site-packages/httpx/_models.py", line 761, in raise_for_status
    raise HTTPStatusError(message, request=request, response=self)
httpx.HTTPStatusError: Client error '400 Bad Request' for url 'https://bedrock-runtime.us-east-1.amazonaws.com/model/anthropic.claude-3-5-sonnet-20240620-v1:0/invoke'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/path/to/venv/lib/python3.11/site-packages/litellm/main.py", line 380, in acompletion
    response = await init_response
               ^^^^^^^^^^^^^^^^^^^
  File "/path/to/venv/lib/python3.11/site-packages/litellm/llms/bedrock_httpx.py", line 1127, in async_completion
    raise BedrockError(status_code=error_code, message=err.response.text)
litellm.llms.bedrock.BedrockError: {"message":"Malformed input request: #: subject must not be valid against schema {\"required\":[\"messages\"]}#: extraneous key [max_tokens_to_sample] is not permitted, please reformat your input and try again."}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  [redacted]
    [redacted -- call to acompletion]
  File "/path/to/venv/lib/python3.11/site-packages/litellm/utils.py", line 1500, in wrapper_async
    raise e
  File "/path/to/venv/lib/python3.11/site-packages/litellm/utils.py", line 1315, in wrapper_async
    result = await original_function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/venv/lib/python3.11/site-packages/litellm/main.py", line 408, in acompletion
    raise exception_type(
          ^^^^^^^^^^^^^^^
  File "/path/to/venv/lib/python3.11/site-packages/litellm/utils.py", line 7631, in exception_type
    raise e
  File "/path/to/venv/lib/python3.11/site-packages/litellm/utils.py", line 6380, in exception_type
    raise BadRequestError(
litellm.exceptions.BadRequestError: litellm.BadRequestError: BedrockException - {"message":"Malformed input request: #: subject must not be valid against schema {\"required\":[\"messages\"]}#: extraneous key [max_tokens_to_sample] is not permitted, please reformat your input and try again."}
```

**Comment by Manouchehri** [comment]
Confirmed on my end as well, #4484 fixed this for me.

**Comment by ishaan-jaff** [comment]
fixed view #4484

**Comment by ishaan-jaff** [comment]
Hi @ubirdio any chance we can hop on a call ? I'd love to learn how how we can improve litellm for you.

Sharing a link to my cal for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat
My linkedin if you prefer DMs https://www.linkedin.com/in/reffajnaahsi/
